### DISCUSSION TITLE
Unable to update to latest release (10.291)

### DISCUSSION THREAD

**Discussion by Nix151** [felddy/foundryvtt-docker#614] [Q&A]
Hi,

I am having issues updating to the latest docker image. I have run the update command (which was successful) but my foundry version has not updated. Tried running the update command again (or with different image tags) but docker just says "Status: Image is up to date for felddy/foundryvtt:release" - any ideas on what I can do to get it updated?

Thanks in advance

**Comment by felddy** [comment]
Make sure you have not inadvertently setting `FOUNDRY_VERSION` to the previous version.  I expect you will see warnings in the log to this effect.  

If unsetting `FOUNDRY_VERSION` was not the solution please update your post and append the log output with `CONTAINER_VERBOSE` set to `true`.  

See:
- https://github.com/felddy/foundryvtt-docker/discussions/208
- https://github.com/felddy/foundryvtt-docker/discussions/451
- https://github.com/felddy/foundryvtt-docker/discussions/291
